### PR TITLE
repeated rides notification + updating head with icon and title

### PR DESCRIPTION
**Pull request by mly32**
### Summary 

- Backend sends notifications when repeated rides are edited. 
- Head icons and title were updated

### Breaking Changes  

<!-- - Database schema change (anything that changes DynamoDB document structure)
<!-- - Other change that could cause problems (Detailed in notes)

### CODE REVIEW

File: server/router/ride.ts [hunk 1]
```diff
@@ -276,20 +258,23 @@ router.put('/:id/edits', validateUser('User'), (req, res) => {
...
-            res.send(editRide);
+            notifyEdit(editRide, change, userType, userId)
+              .then(() => res.send(editRide)).catch(() => res.send(editRide));
+            // res.send(editRide);
           });
         });
       } else {
-        res.send(ride);
+        notifyEdit(ride, change, userType, userId)
+          .then(() => res.send(ride)).catch(() => res.send(ride));
```
Comment (mg2239) [inline server/router/ride.ts:268]: this case doesn't need a notification, as this is technically seen as removing an unscheduled ride

**Review by mg2239 — CHANGES_REQUESTED** [review]
havent tested yet, just did a quick look through. could you also add a notifyEdit for cancelling a scheduled ride (line 302-304) too?

File: server/router/ride.ts (hunk 1 above)
Comment (mly32) [reply server/router/ride.ts]: Updated with changes

File: frontend/package.json [hunk 2]
```diff
@@ -59,5 +59,8 @@
       "last 1 safari version"
     ]
   },
-  "proxy": "https://carriage-web.herokuapp.com"
+  "proxy": "https://carriage-web.herokuapp.com",
+  "//": {
```
Comment (mg2239) [inline frontend/package.json:63]: whats this for?

**Review by mg2239 — APPROVED** [review]
looks good! just a quick comment

**Pull request merged**